core/ecschema-metadata/src/Metadata/KindOfQuantity.ts

```diff
@@ -34,7 +37,9 @@ export class KindOfQuantity extends SchemaItem {
   /** A list of presentation formats. */
   public get presentationFormats(): Array<Format | OverrideFormat> { return this._presentationFormats; }
 
+  /** Persistence unit */
   public get persistenceUnit(): LazyLoadedUnit | LazyLoadedInvertedUnit | undefined { return this._persistenceUnit; }
+  /** @internal */
   protected set persistenceUnit(value: LazyLoadedUnit | LazyLoadedInvertedUnit | undefined) { this._persistenceUnit = value; }
```

Comment:
Our usual pattern is to have a setSomething method instead of a setter property. I guess we can apply that pattern here which then also allow us to mark the set method as internal and the getter as public.

Reply:
That sounds good to me